css-backgrounds-3/Overview.bs

```diff
@@ -2504,7 +2505,8 @@ Layering, Layout, and Other Details</h4>
   and the <i>inner shadows</i> of an element are drawn immediately above the background of that element
   (below the borders and border image, if any).
 
-  <p>If an element has multiple boxes, all of them get drop shadows,
+  <p>Drop shadows are only applied to the [=principal box=].
+  If that box has multiple [=fragments=], then all of them get shadows,
```

Comment:
```
  <p>Unless otherwise specified, drop shadows are only applied to the [=principal box=].
  If the affected box has multiple fragments,
  the shadows are applied as specified in 'box-decoration-break'.
```

Reply:
Done.